src/main/java/org/folio/rest/impl/ExpenseClassAPI.java

```diff
@@ -30,45 +27,38 @@ public ExpenseClassAPI(Vertx vertx, String tenantId) {
 
   @Override
   @Validate
-  public void getFinanceStorageExpenseClasses(String query, @Min(0) @Max(2147483647) int offset, @Min(0) @Max(2147483647) int limit,
-      @Pattern(regexp = "[a-zA-Z]{2}") String lang, Map<String, String> okapiHeaders,
```

Comment:
Did we delet validation here because validation already works by getting min and max values from raml file even without these annotations?

Reply:
I encountered the same validation related problem for migration modules:

_20:52:14 [] [new_tenant] [] [mod_finance_storage] INFO  LogUtil              127.0.0.1:44288 GET /finance-storage/expense-classes/1bcc3247-99bf-4dca-9b0f-7bc51a2998c2 null HTTP_1_1 400 352 -1 tid=null Bad Request HV000151: A method overriding another method must not redefine the parameter constraint configuration, but method ExpenseClassAPI#getFinanceStorageExpenseClasses(String, String, int, int, Map, Handler, Context) redefines the configuration of FinanceStorageExpenseClasses#getFinanceStorageExpenseClasses(String, String, int, int, Map, Handler, Context)._

and then verified that every request first meets with autogenerated interfaces' methods, so has to pass interfaces validation first. These validations are already done in interface methods:
![image](https://user-images.githubusercontent.com/112848437/229456224-b473b641-ea67-4d64-9c4c-f721939db300.png)